src/core/datatypes/forte_string.cpp

```diff
@@ -68,7 +68,7 @@ int CIEC_STRING::fromString(const char *paValue) {
         i += 2;
         continue;
       } else {
-        char specialSymbol;
+        char specialSymbol = '\0';
```

Comment:
`\0` is dangerouse here. Because with that you end the string earlier then it should. Do we have a more neutral character?

Reply:
I wouldn't potentially return a valid string. The method should terminate and indicate an error. After all the string was not valid